zerver/openapi/markdown_extension.py

```diff
@@ -129,6 +129,8 @@ def extract_code_example(
 def render_python_code_example(
     function: str, admin_config: bool = False, **kwargs: Any
 ) -> List[str]:
+    if function not in zerver.openapi.python_examples.TEST_FUNCTIONS:
+        return []
```

Comment:
Do we consider it a bug to have any endpoint without a Python code example?  If so, we could consider `raise AssertionError()` here, rather than `return []`, so that it's not a silent failure.

Reply:
Currently, the two fetch-api-keys endpoints don't have python examples -- hence the empty string.

Reply:
OK, cool.  I think we probably want to add them and then make this obligatory in a follow-up PR.